### ISSUE TITLE
Suricata modul coudnt find files-json.log log file 

### ISSUE THREAD

**Issue by belia2** [cuckoosandbox/cuckoo#2323]
Hello,
the following issue is maybe not even one .. i guess its more my fault by not finiding files or informations were files supposed to be. 

##### My issue is:
when i run a cuckoo analysis anything is good until the analysis is done - then suricata throws an error that it cannot find the 'files-json.log' (me either with locate and find) but anything else runns fine. this have no impact on anything else so it is not a critical situation =)

```
13/6/2018 -- 12:55:24 - <Notice> - This is Suricata version 4.0.4 RELEASE
13/6/2018 -- 12:55:30 - <Notice> - all 13 packet processing threads, 4 management threads initialized, engine started.
13/6/2018 -- 12:55:30 - <Notice> - Signal Received. Stopping engine.
13/6/2018 -- 12:55:30 - <Notice> - Pcap-file module read 261 packets, 54000bytes
2018-06-13 12:55:30,560 [cuckoo.processing.suricata] WARNING: Unable to find the files-json.log log file
```
in the /storage/analysis/7 dir. i found the files.json but no '.log'. 
in  ...../logs/   there where 4 *.bson files
in ......./suricata/   i saw the eve.json and two log files "fast.log" and "stats.log" 

##### My Cuckoo version and operating system are:
OS: Linux Debian 8 (Jessie) 
Cuckoo v.: 2.0.5

##### The log, error, files etc can be found at:
i did not touch the processing conf on the [Suricata] section except from enabled: 'no' to 'yes'
the cuckoo.log logfile (storage/analysis/7/)dont show relevant information about that only one line:

`2018-06-13 12:55:30,560 [cuckoo.core.plugins] DEBUG: Executed processing module "Suricata" for task #7`

cuckoo.log in ./cuckoo/logs:
`2018-06-13 12:55:30,560 [cuckoo.processing.suricata] WARNING: Unable to find the files-json.log log file`

on wish i can post the whole logs but i think they have no more relevant data

TL;DR;
where supposed to be the "files-json.log"...? :
```
suricata = /usr/bin/suricata               # exsists
conf = /etc/suricata/suricata.yaml     # exsists
eve_log =  eve.json                          #exsists in: /.cuckoo/storage/analysis/[any number]/suricata/
files_log = files-json.log
files_dir = files                                 # is this the /.cuckoo/storage/analysis/[any number]/files/ dir?

```

I would like to thank you in advance for your time.
best regards 
Jack

**Comment by RicoVZ** [comment]
Hi belia2,

Thanks for posting an issue.

I think the missing file is only generated by Suricata when enabled in the `suricata.yaml`.
Can you check your `/etc/suricata/suricata.,yaml` to see if `file-log` is enabled?

The config section should look like:
```
- file-log:
      enabled: yes
      filename: files-json.log
```

**Comment by belia2** [comment]
oh damn - that's embarrassing ... that's so obvious that i missed it
Thanks buddy =) 

well... like i guessed it - not a real issue but maybe it helps other people like me xD
in case that this will happen: 
after you fixed it in the suricata.yaml 
cuckoo will tell you again that something is not right 
`YYYY-MM-DD HH:MM:SS,sss  [cuckoo.processing.suricata] WARNING: Suricata files dir is not available. Maybe you forgot to enable Suricata file-store ?`
then just edit the same file like before (suricata.yaml) press ctrl + W (^W) and type "file-store" there you have to enable it  and anything should be fine

**Comment by phamanh1652** [comment]
hi belia2, could i ask if your files-json.log has data because when i cat files-json.log, it is empty (i configured in suricata.yaml (enable=yes, append=yes))
thank you so much
